.gitignore

```diff
@@ -9,3 +9,5 @@ fixspecs/FIX40.xml
 .idea/.name
 
 .idea/pyfixmsg.iml
+
+venv/*
```

Comment:
Well, why add a venv dir here, is it in your own develop environment ? maybe the fixspecs/FIX40.xml could be removed.

Reply:
removed